scripts/stats.py

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env ccp4-python
+
+"""Script to compute some SIMBAD statistics --- work in progress"""
+
+from __future__ import print_function
+
+__author__ = "Felix Simkovic"
+__date__ = "30 Mar 2018"
+__version__ = "1.0"
+
+import os
+import re
+import requests
```

Comment:
In my ccp4-python, requests can't be imported which might cause errors

Reply:
Not sure this is so important right now. This script is really just for us and could be a data dump for database sizes etc ...

Reply:
See my message on slack

Reply:
The urllib2 code would look like this:

req = urllib2.Request(url, data, header)
response = urllib2.urlopen(req)

if response.getcode() == 200:
	count = len(response.readlines())